cmd/podman/system/connection/add.go

```diff
@@ -1,23 +1,19 @@
 package connection
 
 import (
-	"encoding/json"
 	"errors"
 	"fmt"
-	"net"
 	"net/url"
 	"os"
 	"regexp"
 
 	"github.com/containers/common/pkg/completion"
 	"github.com/containers/common/pkg/config"
+	sshCommon "github.com/containers/common/pkg/ssh"
```

Comment:
Do we need the import alias?

Reply:
not anymore, this is a remnant from when golang's ssh package was still around in here